brigade/plugins/tasks/networking/napalm_validate.py

```diff
@@ -0,0 +1,21 @@
+from brigade.core.task import Result
+
+
+def napalm_validate(task, src=None, validation_source=None):
+    """
+    Gather information with napalm and validate it:
+
+        http://napalm.readthedocs.io/en/develop/validate/index.html
```

Comment:
This seems misplaced, it looks a bit strange on RTD and we don't have it for the rest of the tasks.